dagmc_stats/DagmcStats.py

```diff
@@ -0,0 +1,253 @@
+    
+    
+    def populate_triangle_data(self, meshset=None, aspect_ratio=True, area=True):
+        """
+        populate triangle areas and triangle aspect ratios
+        inputs
+        ------
+        meshset : set of entities that are used to populate data. Default value
+                  is None and data of the whole geometry will be populated.
+        aspect_ratio : boolean value that determines whether or not to populate
+                  triangle aspect ratio values. Default value is True.
+        area : boolean value that determines whether or not to populate
+                  triangle area values. Default value is True.
+
+        outputs
+        -------
+        none
+        """
+        tris = get_tris(meshset)
+
+        area = []
+        t_a_r = []
+        for tri in tris:
+            side_lengths = list(get_tri_side_length(tri).values())
+            s = .5*(sum(side_lengths))
+            if area:
+                # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
+                t_a = np.sqrt(s * np.prod(s - side_lengths))
+                area.append(t_a)
+            if aspect_ratio:
+                top = np.prod(side_lengths)
+                bottom = 8*np.prod(s-side_lengths)
+                t_a_r.append(top/bottom)
+        if area:
+            df_tri['tri_area'] = area
+        if aspect_ratio:
+            df_tri['t_a_r'] = t_a_r
+            
+        """if area:
+            area = []
+            for tri in tris:
+                side_lengths = list(get_tri_side_length(tri).values())
+                s = .5*(sum(side_lengths))
+                # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
+                t_a = np.sqrt(s * np.prod(s - side_lengths))
+                area.append(t_a)
+            df_tri['tri_area'] = area
+        if aspect_ratio:
+            t_a_r = []
+            for tri in tris:
+                side_lengths = list(get_tri_side_length(tri).values())
+                s = .5*(sum(side_lengths))
+                top = np.prod(side_lengths)
+                bottom = 8*np.prod(s-side_lengths)
+                t_a_r.append(top/bottom)
+            df_tri['t_a_r'] = t_a_r"""
+        return
+        
+
+    def get_tris(self, meshset):
+        """
+        Get triangles of a volume if geom_dim is 3
+        Get triangles of a surface if geom_dim is 2
+        Else get all the triangles
+
+        inputs
+        ------
+        meshset : set of entities that are used to populate data. Default value
+        is None and data of the whole geometry will be populated.
+
+        outputs
+        -------
+        tris : a list of triangle entities
+        """
+        #Todo: should be here or in init()?
+        dim = _my_moab_core.tag_get_data(dagmc_tags['geom_dim'], meshset)[0][0]
+        # get triangles of a volume
+        if dim == 3:
+            entities = _my_moab_core.create_meshset()
+            for surface in _my_moab_core.get_child_meshsets(root_set):
```

Comment:
Should only be children of this mesh set
```suggestion
            for surface in _my_moab_core.get_child_meshsets(meshset):
```